src/app/components/table/table.ts

```diff
@@ -4025,7 +4025,8 @@ export class ResizableColumn implements AfterViewInit, OnDestroy {
 @Directive({
     selector: '[pReorderableColumn]',
     host: {
-        class: 'p-element'
+        class: 'p-element',
+        '[class.p-reorder-column-disabled]': '!isEnabled()'
```

Comment:
Instead of adding class, could you please use style? 

```
    host: {
        class: 'p-element',
        '[style.cursor]':'isEnabled() ? "move" : "default"'
    }
```

Reply:
Changed to style.